<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>514</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Pluto Scope</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QChartView" name="fft_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>1001</width>
      <height>301</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="setting_group">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLineEdit" name="iiouri_edit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>ip:192.168.2.1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="iiouri_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>58</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IIO URI:</string>
     </property>
    </widget>
    <widget class="QLabel" name="freq_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequency:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="freq_edit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhPreferNumbers</set>
     </property>
     <property name="text">
      <string>1420405752</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="Hz_button">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Hz</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="samprate_unit_label">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MHz</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="samprate_edit">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhPreferNumbers</set>
     </property>
     <property name="text">
      <string>2.30</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="MHz_button">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>MHz</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="GHz_button">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>60</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>GHz</string>
     </property>
    </widget>
    <widget class="QLabel" name="samprate_label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Samprate:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="kHz_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>kHz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_button">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>60</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="buffersize_label">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Buffer Size:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bandwidth_edit">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhPreferNumbers</set>
     </property>
     <property name="text">
      <string>0.21</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="agc_check">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>AGC</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="bandwidth_label">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Bandwidth:</string>
     </property>
    </widget>
    <widget class="QLabel" name="buffersize_unit_label">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>60</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Samps</string>
     </property>
    </widget>
    <widget class="QLabel" name="bandwidth_unit_label">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MHz</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="buffersize_edit">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>60</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhPreferNumbers</set>
     </property>
     <property name="text">
      <string>4096</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="info_group">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>781</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Information</string>
    </property>
    <widget class="QCheckBox" name="is_running">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Is Running</string>
     </property>
    </widget>
    <widget class="QLabel" name="freq_info_label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequency:</string>
     </property>
    </widget>
    <widget class="QLabel" name="freq_info">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0 Hz</string>
     </property>
    </widget>
    <widget class="QLabel" name="samprate_info_label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Samprate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="samprate_info">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0 Hz</string>
     </property>
    </widget>
    <widget class="QLabel" name="bandwidth_info_label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Bnadwidth:</string>
     </property>
    </widget>
    <widget class="QLabel" name="bandwidth_info">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> 0 Hz</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="is_agc">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>AGC</string>
     </property>
    </widget>
    <widget class="QLabel" name="ip_info_label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IP Address:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ip_info">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLabel" name="status_info_label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="status_edit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>461</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Uninitialized.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="logo_label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fftfreq_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Freq: </string>
    </property>
   </widget>
   <widget class="QLabel" name="fftfreq_info">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>0 Hz</string>
    </property>
   </widget>
   <widget class="QLabel" name="fftamp_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Amp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fftamp_info">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>0 dB</string>
    </property>
   </widget>
   <widget class="QFrame" name="mouse_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>1001</width>
      <height>301</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>setting_group</zorder>
   <zorder>fft_view</zorder>
   <zorder>info_group</zorder>
   <zorder>logo_label</zorder>
   <zorder>fftfreq_label</zorder>
   <zorder>fftfreq_info</zorder>
   <zorder>fftamp_label</zorder>
   <zorder>fftamp_info</zorder>
   <zorder>mouse_frame</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
